lib/browser/api/menu.ts

```diff
@@ -28,10 +28,10 @@ Menu.prototype._isCommandIdEnabled = function (id) {
   return this.commandsMap[id] ? this.commandsMap[id].enabled : false;
```

Comment:
(note: I left this one alone here because it's fixed as a side-effect of @codebytere's  changes in #48598)